XenAdmin/XenAdmin.csproj

```diff
@@ -14,11 +14,6 @@
     <SignManifests>false</SignManifests>
     <ManifestCertificateThumbprint>4C1DD393E361B8BBCFA1E1D09539C446F88C0A11</ManifestCertificateThumbprint>

```

Comment:
Not related to this PR, but I've just noticed the tags ManifestCertificateThumbprint and ManifestTimestampUrl which exist in some of the solution's projects and are ancient; we've been using `mk\sign.bat` for signing for a while now.

Reply:
I see, would removing both these nodes from each project they are present in be your recommended course of action?